src/components/PeopleFilters.tsx

```diff
@@ -27,66 +66,44 @@ export const PeopleFilters = () => {
       <div className="panel-block">
         <div className="level is-flex-grow-1 is-mobile" data-cy="CenturyFilter">
           <div className="level-left">
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=16"
-            >
-              16
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=17"
-            >
-              17
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=18"
-            >
-              18
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1 is-info"
-              href="#/people?centuries=19"
-            >
-              19
-            </a>
-
-            <a
-              data-cy="century"
-              className="button mr-1"
-              href="#/people?centuries=20"
-            >
-              20
-            </a>
+            {listCenturies.map(centurie => (
+              <SearchLink
+                params={{
+                  centuries: centuries.includes(centurie)
+                    ? centuries.filter(cent => cent !== centurie)
+                    : [...centuries, centurie],
+                }}
```

Comment:
here is the same as in the comment with functions. Move expression to variable on line 70 and use variables in JSX. Please check your code and improve it according to this comment so not only this case but all similar cases in your code)